sdk/communication/communication-phone-numbers/src/models.ts

```diff
@@ -114,7 +121,30 @@ export {
   OperatorNumberType,
 } from "./generated/src/models/index.js";
 
-export { SipRoutingError, SipTrunkRoute } from "./generated/src/siprouting/models/index.js";
+export {
+  SipRoutingError,
+  SipTrunkRoute,
+  RoutesForNumber,
+  SipRoutingGetOptionalParams,
+  ExpandEnum,
+  KnownExpandEnum,
+  Health,
+  Tls,
```

Comment:
The way the new enum are generated is a nightmare from users perspective. Please try adding this directive for autorest into README-SipRouting.md to not generate enums as strings:

```yaml
directive:
  - from: swagger-document
    where: $.definitions[*].properties[*]["x-ms-enum"]
    transform: >
      if ($.modelAsString) {
        $.modelAsString = false
      }
```

And regenerate code.


If autorest complains about enum validation, you can also add to the autorest commands: 
skip-enum-validation: true 


Another option would be to not export those and create a public facing interface manually, but that would be cumbersome.